picard/file_dilaog.py

```diff
@@ -0,0 +1,274 @@
+        self.moveNext.setIconSize(QSize(35,30))
+        self.moveNext.move(.17*wi+35,10)
+        self.moveNext.clicked.connect(self.moveNext_action)
+
+        self.selectBtn = QPushButton("Select It",self)
+        self.selectBtn.resize(100,30)
+        self.selectBtn.move(wi-0.09*wi,10)
+        self.selectBtn.clicked.connect(lambda:self.select_action(True))
+
+        self.selectBtn1 = QPushButton("Open Folder",self)
+        self.selectBtn1.resize(150,30)
+        self.selectBtn1.move(wi-.219*wi,10)
+        self.selectBtn1.clicked.connect(lambda:self.open_action(True))
+
+        self.lineedit = QLineEdit("Search here",self)
+        self.lineedit.resize(250,30)
+        self.lineedit.move(wi-.51*wi,10)
+        #self.lineedit.hide()
+        self.helpbtn = QPushButton(self)
+        self.helpbtn.resize(40,30)
+        self.helpbtn.setIcon(QIcon('../resources/img-src/search24_24.png'))
+        self.helpbtn.setIconSize(QSize(40,30))
+        self.helpbtn.move(wi-.32*wi,10)
+        #self.helpbtn.clicked.connect(self.lineedit.show)
+        self.helpbtn.clicked.connect(self.search_update)
+
+        self.lw=int(.15*wi)
+        self.l1w=wi-self.lw
+        self.listWidget = QListWidget(self)
+        self.listWidget.resize(self.lw,he-.195*he)
+        self.listWidget.move(0,0)
+
+        self.table = QTableWidget(self)
+        self.table.setColumnCount(3)
+        self.table.setHorizontalHeaderLabels(("Name;Size;Modified").split(";"))
+        self.table.verticalHeader().setVisible(False)
+        self.table.setEditTriggers(QAbstractItemView.NoEditTriggers)
+        self.table.setSelectionBehavior(QAbstractItemView.SelectRows)
+        cw=int(.7*self.l1w)
+        cw1=int(.143*self.l1w)
+        cw2=int(.143*self.l1w)
+        self.table.setColumnWidth(0,cw)
+        self.table.setColumnWidth(1,cw1)
+        self.table.setColumnWidth(2,cw2)
+        self.show_files(False)
+        self.table.resize(self.l1w,he-.26*he)
+        self.table.move(self.lw,50)
+        self.table.setShowGrid(False)
+        self.table.doubleClicked.connect(self.default_setting)
+
+        self.checkBox = QCheckBox("Show hidden files/folders",self)
+        self.checkBox.resize(200,30)
+        self.checkBox.move(wi-.365*wi,he-.17*he)
+        self.checkBox.toggled.connect(lambda:self.show_hidden(self.checkBox))
+
+        self.cb = QComboBox(self)
+        self.cb.addItem("All Supported Formats")
+        self.cb.addItems(self.formats_show)
+        self.cb.resize(200,30)
+        self.cb.move(wi-.183*wi,he-.17*he)
+        self.cb.currentIndexChanged.connect(self.format_modifier)
+
+        self.resize(wi,he)
+        sys.exit(self.exec_())
+
+    def format_modifier(self,i):
+
+        self.formats.clear()
+        self.formats_lower.clear()
+        if not self.cb.itemText(i) == "All Supported Formats":
+            self.formats = [self.cb.itemText(i)]
+        else:
+            self.formats=["MP3","FLAC","OGG","M4A","WMA","WAV","MP4","AAC","flv"]
```

Comment:
Be consistent and respect PEP 8: `self.formats = [...]` is correct, `self.formats=[...]` isn't.